dev-util/ctags/ctags-20170704.ebuild

```diff
@@ -34,8 +34,6 @@ DEPEND="
 	app-arch/unzip
 "
 
-S="${WORKDIR}/${PN}-${COMMITHASH}"
```

Comment:
Since S is easily predictable here, and you have the commit hash already, there's no real advantage from using vcs-snapshot. The eclass is more intended for the cases where you don't store the hash in ebuild.